### DISCUSSION TITLE
Estonian language translation edit/improvement

### DISCUSSION THREAD

**Discussion by reinuke** [home-assistant/frontend#20074] [Localization] [answered]
Hi.. 

How should I go about for suggesting a translation edit? 

Thusfar I've identified two errors with Translations:
"Entity"  -  currently  "Olem" (or "Olemid" for plurral)  would work much better as  "Objekt"  as Object
"Targets" - currently  "Eesmärgid"  (in "Dev.. tools" > "Services" tab) is just wrong. "Eesmärgid" translates into "Goals" and not "Targets" 

Is there a ET (Estonian) language file I could cross-refrence so I could put together a propper list for edits? 
I tried searching the Frontend repo on GitHub but it keeps giving me allsorts of things: 
![image](https://github.com/home-assistant/frontend/assets/157743108/024d3935-4189-47ac-b2eb-200de990ef0c)
Are language files not included with Frontend repo?  Are they somewhere else? 
I'm new to the whole GitHub thing :)

**Comment by karwosts** [comment, marked as answer]
All translations are handled through Lokalise. Feel free to sign up and you can contribute edits to your language.

https://developers.home-assistant.io/docs/translations/

**Comment by reinuke** [comment]
> All translations are handled through Lokalise. Feel free to sign up and you can contribute edits to your language.
> 
> https://developers.home-assistant.io/docs/translations/

Thank you for the lightning fast reply. 
This info should also be pinned somewhere :D

**Comment by juhanjuku** [comment]
I hope it's not big crime if I continue in Estonian  about finesses of our small language

Tundub, et siin osaleb heast tõlkest huvitatud tüüpe. Ma ei teagi kui paljud kasutavad HAd eesti keeles, aga minu huvi tekkis siis kui Voice assistant arenema hakkas. Kodus kõva häälega tahaks rääkida ikka eesti keeles.
Praegu arutleks ikka veel "Entity" teemal. Olem tuleks muidugi asendada, aga "objekt" ei ole ka päris õige sõna.
HAs on rida: Integrations > Devices > Entities. See loogika on suuremast väiksemaks ja Entity on kõige väiksem tükk mis ise enam ei jagune. "Objekt" on eesti keeles pigem mingi suurem kogum: ehitusobjekt, progemises objekt, jne
Entity peaks viitama ikka mingile väikesele osakesele. "element" oleks tähenduselt lähemal kui objekt

Kasutajaliidese tõlkes häirib mind ka "Integration" -> "Sidumised"    Mina pakuks "Liidesed"

@reinuke teises threadis kutsusid kuidagi osalema ka @gerdo ja @LauriLepik
Ma meelitaks siia ka kokku kõik, kes on tõlkimisega tegelenud, aga ma ei oska seda Tagi teha

**Reply by reinuke** [reply]
Those are fighting words @juhanjuku 

Ma valiks lahinguväljaks Facebooki, https://www.facebook.com/share/oEXUj7HDzj8qppQZ/ 

Tee poll ja vaatame mida teised arvavad..

**Reply by reinuke** [reply]
Objekt on mingi **asi** mida sa saad vaadelda, käidelda, muuta, kirjeldada jms..

**Reply by reinuke** [reply]
https://www.facebook.com/share/aChUJxnpnA5EUdUV/
Tegin ise @juhanjuku
